src/utils/test/mocks/contentful/entry.ts

```diff
@@ -1,44 +0,0 @@
-import { Entry, EntryProps, KeyValueMap, SysLink } from 'contentful-management';
```

Comment:
I'd maybe leave all the contentful tests since we're using contentful still. This is probably safe to keep.